src/public/WRLDPointOnRouteResult.h

```diff
@@ -34,6 +34,15 @@ NS_ASSUME_NONNULL_BEGIN
 /// Route Section that the projected point lies on.
 @property (nonatomic, retain, readonly) WRLDRouteSection* routeSection;
 
+/// Route Step Index that the projected point lies on.
+@property (nonatomic, readonly) int routeStepIndex;
+
+/// Route Step Index that the projected point lies on.
+@property (nonatomic, readonly) int routeSectionIndex;
```

Comment:
Shouldn't this be route section index in the comment?
Also probably better to reword as 
`Index for the Route Section that the projected point lies on.`
Same for route step index it can be
`Index for the Route Step that the projected point lies on.`